rt/rs/client/src/main/java/org/apache/cxf/jaxrs/client/AbstractClient.java

```diff
@@ -440,8 +440,7 @@ protected ResponseBuilder setResponseBuilder(Message outMessage, Exchange exchan
             return currentResponseBuilder;
         }
         
-        String reasonPhrase = (String)MessageUtils.getContextualProperty(
-                   responseMessage, HTTPConduit.HTTP_RESPONSE_MESSAGE, null);
+        final String reasonPhrase = (String)responseMessage.get(HTTPConduit.HTTP_RESPONSE_MESSAGE);
```

Comment:
@ffang if you could take a look please, it seems like we should fetch the `HTTPConduit.HTTP_RESPONSE_MESSAGE` from the message itself, not the contextual property, thank you.